setup_dynamixel_node:
  ros__parameters:
    device_name: "/dev/ttyUSB0"
    addr_torque_enable: 24
    protocol_version: 1.0
    baudrate: 1000000

position_control_node:
  ros__parameters:
    device_name: "/dev/ttyUSB0"
    addr_present_position: 36
    addr_goal_position: 30
    protocol_version: 1.0
    baudrate: 1000000

pos_torque_control_node:
  ros__parameters:
    device_name: "/dev/ttyUSB0"
    addr_present_position: 36
    addr_torque_mode_enable: 70
    addr_goal_torque: 71
    protocol_version: 1.0
    baudrate: 1000000